test/distributed/test_distributed.py

```diff
@@ -2250,6 +2250,28 @@ def test_SyncBatchNorm_process_group(self):
         process_group_sync = res50_model_sync.layer1[0].bn1.process_group
         self.assertEqual(process_group_sync, process_group)
 
+    @unittest.skipIf(BACKEND != 'nccl' and BACKEND != 'gloo',
+                     "Only Nccl & Gloo backend support DistributedDataParallel")
+    @skip_if_no_cuda_distributed
```

Comment:
Just FYI, there could be merge conflict with #38078

Reply:
Thanks for the heads-up, assuming that one lands first I'll rebase this.

Reply:
Hey @HapeMask, #38078 has landed and `skip_if_no_cuda_distributed` has been removed on master, could you please rebase this PR. Thanks!

Reply:
Done!